docs/modules/ROOT/pages/migration.adoc

```diff
@@ -29,7 +29,7 @@ RAGStack contains the below packages as of version `0.7.0`. When RAGStack is ins
 
 == Example LangChain migration
 
-Here is a simple LangChain application that loads a dataset from HuggingFace and embeds the document objects in AstraDB.
+Here is a simple LangChain application that loads a dataset from HuggingFace and embeds the document objects in Astra DB.
```

Comment:
Should this specify "Astra DB Serverless"? A renamed version of Astra DB Classic is coming back, so it may be important to specify which version of Astra DB applies.

Reply:
I agree @KimberlyFields. When I use the updated attribute {db-vector}, it will render as 

"...and embeds the document objects in Astra DB Serverless (Vector)." 

"Astra DB Serverless (Vector)" is an example of awkward product naming, but I think it's what users see elsewhere in the documentation.  Right?

Reply:
Yes. Updated in next commit.